src/nvc.c

```diff
@@ -291,10 +252,27 @@ static int elaborate(int argc, char **argv)
          opt_set_str(OPT_DUMP_VCODE, optarg ?: "");
          break;
       case 'c':
-         if (optarg)
-            cover_mask = parse_cover_mask(optarg);
+         if (optarg) {
+            char LOCAL *wc_buf[1] = {
+               xcalloc(32)
```

Comment:
There's nothing stopping the user putting more than 32 characters after `ignore-arrays-from-`.

Reply:
I added protection into `opt_parse_comma_separated`.